sdks/python/apache_beam/yaml/integration_tests.py

```diff
@@ -124,7 +124,10 @@ def provider_sets(spec, require_available=False):
             for p in spec['pipelines']))
 
   def filter_to_available(t, providers):
-    if require_available:
+    if t == 'LogForTesting':
+      # Don't fan out to all the (many) possibilities for this one...
+      return [p for p in providers if isinstance(p, yaml_provider.InlineProvider)]
```

Comment:
Maybe we could do something like `[p for p in providers if isinstance(p, yaml_provider.InlineProvider)] + [p for p in providers if not isinstance(p, yaml_provider.InlineProvider)][:1]` to get both an inline and non-inline provider?

Reply:
This would still double the number of tests we need (and make the number of tests change based on whether one added logging) I'm going to leave it as is (though I like the idea of explicitly limiting to 1).